soc/arm/microchip_mec/Kconfig

```diff
@@ -11,6 +12,210 @@ config SOC_FAMILY
 	string
 	default "microchip_mec"
 
+menuconfig MCHP_MEC_HEADER
+	bool "The output binary with MCHP MEC binary header"
+	help
+	  On Microchip MEC series chip, the ROM code loads firmware image from flash
+	  to RAM using a TAG to locate a Header which specifies the location and
+	  size of the firmware image. Enable this to invoke the mec_spi_gen tool
+	  which generates an SPI image with TAG, Header, and firmware binary. This
+	  tool does not produce a signed image which can be authenticated by the
+	  Boot-ROM. Use the full Microchip SPI image generator program for
+	  authentication and all other Boot-ROM loader features. Refer to the MCHP
+	  EVB boards for an example.
```

Comment:
Can you elaborate when one should use the `mec_spi_gen` tool vs. the full SPI image generator program `mec172x_spi_gen_lin_x86_64`?

For instance, is there a pin strapping, or a different part number used to run signed and unsigned images? 

Also, if this option can only create unsigned images, it could be clearer to the user if the name was `MCHP_MEC_UNSIGNED_HEADER`

Reply:
`mec_spi_gen` tool should be turned ON only during the chromium build purpose, that's the current usage. 

will rename to `MCHP_MEC_UNSIGNED_HEADER` for clarity.

Reply:
renamed